contributors/devel/conformance-tests.md

```diff
@@ -1,57 +1,69 @@
+SIG Architecture has approved to define the core set of interoperable 
+features that all Kubernetes clusters must support. The tests verify
+that the expected functionality works as a user might encounter it in
+the wild.
 
-Contributors must write and submit e2e tests first (approved by owning Sigs). 
-Once the new tests prove to be stable in CI runs, later create a follow up PR 
-to add the test to conformance. This approach also decouples the development 
-of useful tests from their promotion to conformance.
+The process to add new conformance tests is intended to decouple the
+development of useful tests from their promotion to conformance:
+
+- Contributors write and submit e2e tests, to be approved by owning SIGs
+- Tests are proven to meet the conformance test requirements
+- A follow up PR is submitted to promote the PR to conformance
 
-A conformance test verifies the expected functionality works as a user might encounter it in the wild, 
 and tests should begin by covering the most important and visible aspects of the function.
 
 ### Conformance Test Requirements
 
-A test is eligible for promotion to conformance if it meets the following requirements:
-
-- testing GA feature (not alpha or beta APIs, nor deprecated features)
-- must be portable (not dependent on provider-specific capabilities or on the public internet)
-- cannot test a feature which obviously cannot be supported on a broad range of platforms 
-(i.e. testing of multiple disk mounts, GPUs, high density)
-- cannot test an optional feature (e.g. not policy enforcement)
-- should be non-privileged (neither root on nodes, network, nor cluster)
-- cannot rely on any particular non-standard file system permissions granted to containers or users 
-(i.e. sharing writable host /tmp with a container)
-- should be stable and run consistently
-- cannot skip providers (there should be no Skip like directives added to the test), 
-especially in the Nucleus or Application layers as described 
-[here](https://github.com/kubernetes/community/blob/master/contributors/devel/architectural-roadmap.md).
-- cannot test cloud provider specific features (i.e. GCE monitoring, S3 Bucketing, ...)
-- should work with default settings for all configuration parameters 
-(example: the default list of admission plugins should not have to be tweaked for passing conformance).
-- cannot rely on any binaries that are not required for the
-linux kernel or for a kubelet to run (i.e. git)
+Conformance tests should be boring. They should rely on nothing special, test only
+features that are available everywhere, and generate the same results every time.
+
+More specifically, a test is eligble for promotion to conformance if:
+
+- it tests only GA, non-optional features or APIs (ie: no alpha or beta endpoints, 
+  no feature flags required)
+- it works for all providers (ie: no `SkipIfProviderIs`/`SkipUnlessProviderIs` calls)
+- it is non-privileged (ie: no root on nodes, network, nor cluster)
```

Comment:
That's not true^

Reply:
How?  There are conformance tests today that require privileges?  Or are you saying that there's functionality that is core that requires privileges to verify that we haven't written tests for yet?

Reply:
There are tests that require elevated privs already b/c they are in the API.